src/App/Pages/VaultFilterViewModel.cs

```diff
@@ -64,15 +65,21 @@ public string GetVaultFilterOrgId()
         protected async Task InitVaultFilterAsync()
         {
             _organizations = await organizationService.GetAllAsync();
-            ShowVaultFilter = await policyService.ShouldShowVaultFilterAsync();
-            if (ShowVaultFilter)
+            if (_organizations?.Any() ?? false)
             {
-                _hideMyVaultFilterOption = await policyService.PolicyAppliesToUser(PolicyType.PersonalOwnership);
-                if (_vaultFilterSelection == null)
+                _personalOwnershipPolicyApplies = await policyService.PolicyAppliesToUser(PolicyType.PersonalOwnership);
+                _singleOrgPolicyApplies = await policyService.PolicyAppliesToUser(PolicyType.OnlyOrg);
```

Comment:
`_singleOrgPolicyApplies` doesn't need to be a field, it can be a var or directly put the await call inside the if